src/core/song.cpp

```diff
@@ -1323,6 +1327,8 @@ Song::FileType Song::FiletypeByMimetype(const QString &mimetype) {
   if (mimetype.compare("audio/x-spc"_L1, Qt::CaseInsensitive) == 0) return FileType::SPC;
   if (mimetype.compare("audio/x-vgm"_L1, Qt::CaseInsensitive) == 0) return FileType::VGM;
 
+  // XXX: ALAC? Does gstreamer just give back audio/mpeg like it does for MP3 and AAC?  --paper
```

Comment:
Think it should be `audio/x-alac`, but please test a file and check.

Reply:
Yep, that's right.